interface/main/messages/print_postcards.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * /library/MedEx/print_postcards.php
+ *
+ * This file is executed as a background service
+ * either through ajax or cron.
+ *
+ * Copyright (C) 2017 MedEx <magauran@MedExBank.com>
+ *
+ * LICENSE: This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  Portions of this were developed using Terry Hill's addr_label code.
+ *
+ * @package OpenEMR
+ * @author MedEx <support@MedExBank.com>
+ * @link http://www.open-emr.org
+ */
+
+require_once("../../globals.php");
+require_once("$srcdir/classes/fpdf/fpdf.php");
+
+# This is based on session array. 
+$pid_list = array();
+$pid_list = $_SESSION['pidList'];
+
+$pdf = new FPDF('L', 'mm', array(148,105));
+$last = 1;
+$pdf->SetFont('Arial','',14);
+
+$sql = "SELECT * FROM facility ORDER BY billing_location DESC LIMIT 1";
+$facility = sqlQuery($sql);
+
+$sql = "select * from medex_prefs";
+$prefs =  sqlQuery($sql);
+if ($prefs['postcard_top']) {
+    $postcard_top = $prefs['postcard_top'];
+} else {
+    $postcard_top ='';
+}
+
+$postcard_message = $postcard_top."\n".xlt('Please call our office to schedule')."\n".xlt('your next appointment at')." ".text($facility['phone']).".
+	\n\n   ".text($facility['street'])."\n   
+	".text($facility['city']).", ".text($facility['state'])."  ".text($facility['postal_code']);
```

Comment:
Since this is going directly to pdf, would not html escape this stuff:
``````
$postcard_message = $postcard_top."\n".xl('Please call our office to schedule')."\n".xl('your next appointment at')." ".$facility['phone'].".
	\n\n   ".$facility['street']."\n   
	".$facility['city'].", ".$facility['state']."  ".$facility['postal_code'];